### ISSUE TITLE
Update Link to Zenodo

### ISSUE THREAD

**Issue by mwakok** [SATAY-LL/Transposonmapper#1]

**Comment by leilaicruz** [comment]
do you mean in Zenodo itself? like editing the metadata is now describing the project? 

should we make a new release with this new organization?

**Comment by mwakok** [comment]
Did you create an link between Zenodo and Github before? You can set this up so that each time we release a new version on the main branch, Zenodo automatically uploads it: https://guides.github.com/activities/citable-code/. If you had done this already, you would need to update the link as it would now point to the old repository

**Comment by leilaicruz** [comment]
I would need to grant third party access to Zenodo to the organization in github, to enable the new repository to be visible for zenodo . But I am wondering I think Zenodo will create a release from the main branch of the SATAY-LL  once this is changed , isnt it?

**Comment by mwakok** [comment]
Yes, the third party access is required.

I believe Zenodo will only upload a new(!) release, it won't do anything upon push or pull requests or with any existing releases. As long as we don't create a new release ourselves, Zenodo won't upload anything

**Comment by leilaicruz** [comment]
so it was a bit hidden and tricky how to grant access to the organization frm my personal account but I managed , so now Zenodo sees the repos in the organization . I see that the doi automatically was updated to the Laan -Lab-satay-analysis repo , and because transposonmapper is a new one , we would need to make a new release out of this one. 
![image](https://user-images.githubusercontent.com/11459658/124898140-34b65e00-dfdf-11eb-96e2-e2631bda8dea.png)

Should I update the doi with the old repo ?

**Comment by mwakok** [comment]
What do you mean with "update the doi with the old repo"?

**Comment by leilaicruz** [comment]
so if I  get you right, now our doi is pointing to the old repo right? to here https://github.com/SATAY-LL/LaanLab-SATAY-DataAnalysis. This doi is in fact the same as the one we have currently because the repo was totall transferred to this one. 

My understanding is that if you want an update on the zenodo metadata with the transposonmapper repo , we need to create a new release from this one and update what we have there, so that it follows the webhook from the transposonmapper new repo, isnt it?

**Comment by mwakok** [comment]
Indeed, the DOI is only pointing to the archive on Zenodo, but it's metadata links it to the old repo. The main aim is to ensure that new releases from the new transposonmapper repo are archived as new versions of the old DOI (Zenodo can provide a DOI that will always point to the latest version). I haven't done that before, so it could be that we need to update the archive on Zenodo manually the first time.

Again, we can also figure this out next week. The problem with Zenodo (I believe) is that only one user has admin access to the archive. This is why it's best if the archive is controlled by you, but it also means I can't verify the process myself.

**Comment by leilaicruz** [comment]
yes , I think it doesnt allow  to update the existing version to the new repo , we should instead create a new release from https://github.com/SATAY-LL/Transposonmapper and then all the further releases are automatically tracked by Zenodo. Because the metadata is linked to the repository where the release came from.
